benchmark/util/style-text.js

```diff
@@ -9,10 +9,11 @@ const bench = common.createBenchmark(main, {
   messageType: ['string', 'number', 'boolean', 'invalid'],
   format: ['red', 'italic', 'invalid'],
   validateStream: [1, 0],
+  noColors: [1, 0],
```

Comment:
```suggestion
  noColors: ['1',  ''],
```

Reply:
simple question why ? on `validateStream` you use `1` & `0`

Reply:
The env variable value is a string, not a boolean nor a number

Reply:
oh got it tanks !